statix.lang/metaborg.yaml

```diff
@@ -1,12 +1,13 @@
 ---
-id: org.metaborg:statix.lang:${metaborgVersion}
+id: org.metaborg:statix2.lang:${metaborgVersion}
 name: StatixLang
 metaborgBaselineVersion: 2.5.16
 dependencies:
   compile:
   - org.metaborg:org.metaborg.meta.lang.esv:${metaborgBaselineVersion}
   - org.metaborg:org.metaborg.meta.lang.template:${metaborgBaselineVersion}
   - org.metaborg:org.metaborg.meta.nabl2.lang:${metaborgVersion}
+  - org.metaborg:org.metaborg:statix.lang:${metaborgVersion}
```

Comment:
I forgot to mention previous week, but if you add `org.metaborg:sdf3.ext.statix:${metaborgVersion}` here, signatures for Statix will be generated automatically. It might require some massaging of the grammar though. See https://www.spoofax.dev/howtos/statix/signature-generator/ for more information.